hack/e2e-suite/services.sh

```diff
@@ -79,6 +79,22 @@ function make_namespace() {
 __EOF__
 }
 
+wait_for_apiserver() {
+  echo "Waiting for apiserver to be up"
+
+  local i
+  for i in $(seq 1 12); do
+    results=($(ssh-to-node "${master}" "
+      wget -q -T 1 -O - http://localhost:8080/healthz || true
```

Comment:
It is using wget in more places (e.g. in busybox there is not curl).
BTW - I'm already working on rewriting this test in GO - see #11056 (hopefully will finish it tomorrow) so I would prefer not spend too much time on this one.